packages/sdk/Dockerfile

```diff
@@ -34,26 +41,28 @@ FROM builder as su-exec
 
 # v0.2 -> f85e5bde1afef399021fbc2a99c837cf851ceafa
 WORKDIR /usr/local/src
-RUN <<EOF
-set -e
-curl -sL https://github.com/ncopa/su-exec/archive/f85e5bde1afef399021fbc2a99c837cf851ceafa.tar.gz | tar --strip-components=1 -zxvf -
-make
-EOF
+ADD --keep-git-dir=true https://github.com/ncopa/su-exec.git#f85e5bde1afef399021fbc2a99c837cf851ceafa /usr/local/src
```

Comment:
any reason for the `--keep-git-dir`?

Reply:
In this su-exec case, no particular reason.

Reply:
Can I remove all `--keep-git-dir` from this Dockerfile?
This and the one from genext2fs

Reply:
What your concern here?

I think you could remove it. 

Some build processes use the git workspace to gather information about versioning and stuff, don't know if that's the case here.

Reply:
I don't think that's the case for theses builds.
Just wanted to remove an explicit option if we don't explicitly need it.